packages/attestation-service/src/requestHandlers/get_attestation.ts

```diff
@@ -36,10 +36,47 @@ class GetAttestationRequestHandler {
     this.key = getAttestationKey(getRequest)
   }
 
-  async getAttestationRecord() {
-    return findAttestationByKey(this.key, {
+  async withAttestationAndSecurityCodeChecked(f: (attestation: AttestationModel) => any) {
```

Comment:
Nit: I personally like shorter variable names, but I believe the js convention is to call these `callback`, especially in more critical functions like this